src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -251,11 +235,27 @@ object types interchangeably.
 References
 ----------
 
--  `Primitive Data Types
-   (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html>`__
--  `Autoboxing and Unboxing
-   (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/data/autoboxing.html>`__
--  `Variables
-   (docs.oracle.com) <https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html>`__
--  `Arrays
-   (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html>`__
+#. `Primitive Data Types (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html>`__
+#. `Autoboxing and Unboxing (docs.oracle.com) <http://docs.oracle.com/javase/tutorial/java/data/autoboxing.html>`__
+#. `Variables (docs.oracle.com) <https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html>`__
+
+Check Your Understanding
+-------------------------
+
+.. admonition:: Question
+
+   Which of the following is not a number data type in Java:
```

Comment:
To emphasize the negative, use all caps for any NOT question.

"Which of the following is NOT a number data type in Java:"